app/services/rapid_ready_for_decision/processor_selector.rb

```diff
@@ -6,38 +6,40 @@ def initialize(form526_submission)
       @form526_submission = form526_submission
     end
 
+    RrdConstants = RapidReadyForDecision::Constants
+
+    DISABILITY_TO_PROCESSOR_CLASS = {
+      hypertension: RapidReadyForDecision::DisabilityCompensationJob,
+      asthma: RapidReadyForDecision::Form526AsthmaJob
+    }.freeze
```

Comment:
Since we're referencing an actual class, I prefer to reference the class so that in case we ever change the class name, the error will be obvious. However, I will concede to your suggestion to get this PR merged.